.github/workflows/release.yaml

```diff
@@ -26,16 +26,16 @@ jobs:
           git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
 
       - name: Install Helm
-        run: |
-          echo "installing helm 3..."
-          curl -sSL https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash
+        uses: azure/setup-helm@v4.0.0
+        with:
+          version: 'v3.14.0'
```

Comment:
hello @duhow, thanks a lot for the contribution! While we can not use Github Actions from 3rd parties according to https://github.com/apache/incubator-devlake-helm-chart/pull/201

Reply:
oh, i see.. 🤔